monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ResultsFolderController.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.datacleaner.monitor.server.controllers;
 
+import java.sql.Timestamp;
```

Comment:
No SQL packages, please. If you something a little more modern that Date and Calendar, there's the Java 8 `java.time` package, which also has good tools for checking if one temporal value is after another.